app/src/main/java/org/schabi/newpipe/player/VideoPlayerImpl.java

```diff
@@ -505,6 +505,9 @@ public boolean onKeyDown(final int keyCode) {
         switch (keyCode) {
             default:
                 break;
+            case KeyEvent.KEYCODE_SPACE:
+                onPlayPause();
```

Comment:
add `if(isFullscreen) ` before onPlayPause please. Just to make sure we don't have any side effects if space bar is hit while not in fullscreen

Reply:
and then squash your commits please so that we just have one :)

Reply:
Done

Reply:
I like the subtle bugfix more than the actual change of this PR: I think we should rename it somehow to make it clear, that it also fixes that bug.

Reply:
I just merged it and added another line to the upcoming changelog. Thank you